ddtrace/contrib/aioredis/patch.py

```diff
@@ -157,7 +155,7 @@ async def traced_13_execute_pipeline(func, instance, args, kwargs):
 
     cmds = []
     for _, cmd, cmd_args, _ in instance._pipeline:
-        parts = [cmd.decode() if isinstance(cmd, bytes) else cmd]
+        parts = [cmd]
         parts.extend(cmd_args)
```

Comment:
Can be simplified to:
```suggestion
        parts = [cmd] + cmd_args
```

Will fix it if the overall consensus on the approach is positive.

Reply:
this approach lgtm

Reply:
Sorry, did you mean the suggestion or current approach? I switched to the suggestion, but can revert back to the original.

Reply:
Nevermind, `cmd_args` is a tuple, so we'd have to convert to a list. `extends` is better.